contract/test/Bank.spec.ts

```diff
@@ -26,88 +26,155 @@ describe('Bank', () => {
     let member1: SignerWithAddress
     let member2: SignerWithAddress
 
-    context('I want to deploy my Bank Smart Contract', () => {
-      before(async () => {
-        ;[owner, contractor, member1, member2] = await ethers.getSigners()
-        await deployContract(owner)
-      })
+    before(async () => {
+      ;[owner, contractor, member1, member2] = await ethers.getSigners()
+      await deployContract(owner)
+    })
 
-      it('Then I become the owner of the contract', async () => {
+    context('Deployment', () => {
+      it('should set the correct owner and tips address', async () => {
         expect(await bankProxy.owner()).to.eq(await owner.getAddress())
+        expect(await bankProxy.tipsAddress()).to.eq(await tipsProxy.getAddress())
       })
```

Comment:
check of all expected behaviour. when deploy a bank contract : Having the right owner and having the right tips Address